_config.yml

```diff
@@ -10,6 +10,9 @@ kramdown:
 
     exclude: ['Gemfile', 'Gemfile.lock', 'Rakefile.rb', 'README.md']
 
+plugins:
+    [jekyll-gist]
```

Comment:
Without this plugin list, I couldn't run `rake clean serve`.
See https://github.com/jekyll/jekyll-gist#installation